### DISCUSSION TITLE
Im not sure what can / cant do with voyager.

### DISCUSSION THREAD

**Discussion by leostereo** [thedevdojo/voyager#5540] [General]
Hi guys , I ve been reading and testing voyager and im  not sure what is it for yet.
Is there any complete and working app  to take as an example ? 
I was trying with this :
https://devdojo.com/scripts/php/geekshop
But can not login / download the code either.
Regards.

**Comment by Marcus5114** [comment]
Hey!
Laravel Voyager is an Admin Packagen, that means that it is in charge of facilitating the task of generating an administration environment for a website (for example, an ecommerce, a user management system, etc.)

So you can almost skip a big step in a development just by installing Voyager and get the user management out of the box from it
